WalletWasabi.Gui/Global.cs

```diff
@@ -140,8 +139,11 @@ public async Task InitializeNoWalletAsync(TerminateService terminateService)
 
 				var userAgent = Constants.UserAgents.RandomElement();
 				var connectionParameters = new NodeConnectionParameters { UserAgent = userAgent };
+				UpdateChecker updateChecker = new(TimeSpan.FromMinutes(7), Synchronizer);
```

Comment:
`UpdateChecker` should be disposed I think.

Reply:
The current concept here is that HostedServices will do.

Reply:
You are right 👍